Katello/katello
# Fixes #28778 - Disable auto-attach for Host Collection

This PR disables the auto attach button in Host Collection while in Simple Content Access.

**To test this PR**:
1. Create a new Organization. 
2. Import a golden ticket enabled manifest. 
3. Create a Host Collection in Simple Content Access.
4. Click on Subscription Management.
5. The auto attach button should be disabled.
